Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+pipeline {
+  agent {
+    docker {
+      image 'andreldsa/nodejs-python'
+      args '--group-add staff --user root'
+    }
+    
+  }
+  stages {
+    stage('Tests') {
+      steps {
+        parallel(
+          "Backend": {
+            sh './ecis test server --clean'
+            
+          },
+          "Frontend": {
+            sh '''
```

Comment:
How will we run the client's tests now?

Reply:
With the command `./ecis test client` in the src of the project.